src/internal_events/events_processed.rs

```diff
@@ -0,0 +1,14 @@
+use crate::internal_events::InternalEvent;
+use metrics::counter;
+
+/// A metric to denote the number of events processed by a topology component.
+/// This is wired up already for transforms in the `topology::builder::build_pieces` function,
+/// so you don't need to do that yourself.
+#[derive(Debug)]
+pub struct EventProcessed;
+
+impl InternalEvent for EventProcessed {
```

Comment:
_nit:_ The file name is plural but the struct name is singular :slightly_smiling_face:

Reply:
Oh thank you.